test/test_jit.py

```diff
@@ -7736,6 +7736,27 @@ def forward(self, x, seq_lens):
         f = io.BytesIO()
         torch.onnx._export(m, (x, seq_lens), f, verbose=False)
 
+    def test_script_pack_padded_sequence(self):
+        from torch.nn.utils.rnn import pack_padded_sequence, pad_packed_sequence
+
+        def pack_padded_pad_packed_script(x, seq_lens):
+            x = pack_padded_sequence(x, seq_lens)
+            x, _ = pad_packed_sequence(x)
+            return x
+
+        T, B, C = 3, 5, 7
+        x = torch.ones((T, B, C))
+        seq_lens = torch.tensor([3, 3, 2, 2, 1])
+        # set padding value so we can test equivalence
+        for b in range(B):
+            if seq_lens[b] < T:
+                x[seq_lens[b]:, b, :] = 0
+
+        eager_seq = pack_padded_pad_packed_script(x, seq_lens)
```

Comment:
reason not to do checkScript here instead ?

Reply:
`CheckScript` is also checking the string frontend and the string frontend could not handle the comments starting with `r`..